conda/utils.py

```diff
@@ -13,6 +13,23 @@
 log = logging.getLogger(__name__)
 stderrlog = logging.getLogger('stderrlog')
 
+def gnu_get_libc_version():
```

Comment:
probably best to add a `@memoize` decorator here (or `@memoized`, whatever it is)